Gopkg.toml

```diff
@@ -54,9 +54,9 @@ required = [
   name = "github.com/giantswarm/errors"
   branch = "master"
 
-[[constraint]]
+[[override]]
   name = "github.com/giantswarm/k8sportforward"
-  branch = "master"
+  branch = "tiller-again"
```

Comment:
This needs to be reverted before merging, right?

Reply:
Yes but these changes still don't work sadly. Sorry I asked for review too early. It worked in ginger.